Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -292,4 +292,14 @@ WK_TRACE_POINTS_OUTPUT_PATH = $(WK_TRACE_POINTS_OUTPUT_PATH_USE_OVERRIDE_FRAMEWO
 WK_TRACE_POINTS_OUTPUT_PATH_USE_OVERRIDE_FRAMEWORKS_DIR_NO_macosx = $(DSTROOT)/AppleInternal/Library/Ariadne/Plists/WebKit.plist;
 WK_TRACE_POINTS_OUTPUT_PATH_USE_OVERRIDE_FRAMEWORKS_DIR_NO_iphoneos = $(DSTROOT)/AppleInternal/Library/Ariadne/Plists/WebKit.plist;
 
+WK_SERVICE_EXTENSIONS_LDFLAGS =
```

Comment:
Nit - it looks like the other linker flag settings are defined before they're used in the definition of `FRAMEWORK_AND_LIBRARY_LDFLAGS`. Can we move this to before `FRAMEWORK_AND_LIBRARY_LDFLAGS` for consistency?